src/isPalindrome.js

```diff
@@ -0,0 +1,6 @@
+export default function isPalindrome(string){
+  var newString = string.replace(/[^\w]_?/g,'').toLowerCase()
```

Comment:
I would recommend using a separate function to describe the regex expression you're using. Regex is notoriously difficult to parse and a function containing an identifier that describes the expected result would be helpful.

For example: 

```javascript
const lowercaseWithoutNonAlphanumeric = str => str.replace(/[^\w]_?/g,'').toLowerCase()
```